components/content/index.css

```
@@ -1,21 +1,42 @@
 .content {
     background-color: var(--blueviolet);
+    border-radius: 12px;
 }
 
 .content-holder {
     display: flex;
     height: 100%;
-    background-image: linear-gradient(var(--red), var(--green));
+    border-radius: 12px;
 }
 
 /* Content aside */
 
 .content-aside {
-    width: 20%;
+    width: 40%;
     min-width: 100px;
-    background-image: url(../../assets/img/aside-background.jpg);
+    background-color: var( --light-gray-tint-1 );
+    display: flex;
+    flex-direction: column;
+}
+
+h1 {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 100%;
+    color: var( --black );
 }
 
+.job-position {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 100%;
+    color: var( --gray );
+
+}
```

```suggestion
}

.content-aside__avatar {
    width: 315px;
    padding-bottom: 40px;
}

.content-aside__avatar img {
    border-radius: 12px;
}
```

This part for the avatar wrapper